common/web/gesture-recognizer/src/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

This gets the right version of bash on macOS.